src/components/Button/Button.css

```diff
@@ -88,3 +88,9 @@
 .y-button--icon-wrapper-right {
   padding-left: 4px;
 }
+
+.y-button--secondary-text {
+  visibility: hidden;
+  overflow: hidden;
+  height: 0px;
```

Comment:
if this doesn't work across browsers we may need to do `height: 1px;` and `margin-top: -1px;`

Reply:
this could also just be `height: 0;` but postcss will handle that